mesa/space.py

```diff
@@ -26,6 +26,7 @@
 from warnings import warn
 
 import numpy as np
+import networkx as nx
```

Comment:
This makes we wonder if we should set `networkx` as an optional dependency. Not everyone uses the `NetworkGrid`. This concern is not a blocker to this PR, however.

Reply:
yes, this can be good in my opinion, but I'm fine also with the current way